From a735c0386bd5f8d3717da834b240bad6706078b2 Mon Sep 17 00:00:00 2001
From: Zhangbin Tong <zebulun.tong@rock-chips.com>
Date: Wed, 5 Apr 2017 10:53:41 +0800
Subject: [PATCH] arm: dts: rockchip: add mpu6050 to rk3288-evb-act8846

The evaluation board using the mpu6050 as the sensor, which
combine a 3-axis gyroscope and a 3-axis accelerometer

Change-Id: Ia41d2c038e38de3cb6d24d81d4b018670935d39a
Signed-off-by: Zhangbin Tong <zebulun.tong@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-evb-act8846.dts | 28 ++++++++++++++++++++++++
 1 file changed, 28 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-evb-act8846.dts b/arch/arm/boot/dts/rk3288-evb-act8846.dts
index 7ac1a78493f4..abd6fb7881c3 100644
--- a/arch/arm/boot/dts/rk3288-evb-act8846.dts
+++ b/arch/arm/boot/dts/rk3288-evb-act8846.dts
@@ -240,6 +240,28 @@
 	};
 };
 
+&i2c1 {
+	status = "okay";
+	clock-frequency = <400000>;
+
+	mpu6050@68 {
+		compatible = "invensense,mpu6050";
+		status = "okay";
+		pinctrl-names = "default";
+		pinctrl-0 = <&mpu6050_irq_gpio>;
+		reg = <0x68>;
+		irq-gpio = <&gpio8 0 IRQ_TYPE_EDGE_RISING>;
+		mpu-int_config = <0x10>;
+		mpu-level_shifter = <0>;
+		mpu-orientation = <0 1 0 1 0 0 0 0 1>;
+		orientation-x= <0>;
+		orientation-y= <1>;
+		orientation-z= <0>;
+		support-hw-poweroff = <1>;
+		mpu-debug = <1>;
+	};
+};
+
 &lvds_panel {
 	power-supply = <&vcc_lcd>;
 };
@@ -260,4 +282,10 @@
 			rockchip,pins = <7 9 RK_FUNC_GPIO &pcfg_pull_none>;
 		};
 	};
+
+	mpu6050 {
+		mpu6050_irq_gpio: mpu6050-irq-gpio {
+			rockchip,pins = <8 0 RK_FUNC_GPIO &pcfg_pull_none>;
+		};
+	};
 };
-- 
2.35.3

